common/fingerprints.py

```diff
@@ -34,9 +34,14 @@ class TOYOTA:
     # sent messages
     0x194: 4, 0x1fa: 8, 0x30c: 8, 0x33d: 5,
   },
-  HONDA.ODYSSEY: {
+  HONDA.ODYSSEY: [{
     57L: 3, 148L: 8, 228L: 5, 229L: 4, 316L: 8, 342L: 6, 344L: 8, 380L: 8, 399L: 7, 411L: 5, 419L: 8, 420L: 8, 427L: 3, 432L: 7, 450L: 8, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 542L: 7, 545L: 6, 597L: 8, 662L: 4, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 806L: 8, 808L: 8, 817L: 4, 819L: 7, 821L: 5, 825L: 4, 829L: 5, 837L: 5, 856L: 7, 862L: 8, 871L: 8, 881L: 8, 882L: 4, 884L: 8, 891L: 8, 892L: 8, 905L: 8, 923L: 2, 927L: 8, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1029L: 8, 1036L: 8, 1052L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1092L: 1, 1108L: 8, 1110L: 8, 1125L: 8, 1296L: 8, 1302L: 8, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1614L: 5, 1615L: 8, 1616L: 5, 1619L: 5, 1623L: 5, 1668L: 5
   },
+  # Odyssey Elite
+  {
+    57L: 3, 148L: 8, 228L: 5, 229L: 4, 304L: 8, 342L: 6, 344L: 8, 380L: 8, 399L: 7, 411L: 5, 419L: 8, 420L: 8, 427L: 3, 432L: 7, 440L: 8, 450L: 8, 463L: 8, 464L: 8, 476L: 4, 490L: 8, 506L: 8, 507L: 1, 542L: 7, 545L: 6, 597L: 8, 662L: 4, 773L: 7, 777L: 8, 780L: 8, 795L: 8, 800L: 8, 804L: 8, 806L: 8, 808L: 8, 817L: 4, 819L: 7, 821L: 5, 825L: 4, 829L: 5, 837L: 5, 856L: 7, 862L: 8, 871L: 8, 881L: 8, 882L: 4, 884L: 8, 891L: 8, 892L: 8, 905L: 8, 923L: 2, 927L: 8, 929L: 8, 963L: 8, 965L: 8, 966L: 8, 967L: 8, 983L: 8, 985L: 3, 1029L: 8, 1036L: 8, 1052L: 8, 1064L: 7, 1088L: 8, 1089L: 8, 1092L: 1, 1108L: 8, 1110L: 8, 1125L: 8, 1296L: 8, 1302L: 8, 1600L: 5, 1601L: 8, 1612L: 5, 1613L: 5, 1614L: 5, 1616L: 5, 1619L: 5, 1623L: 5, 1668L: 5, 416944369L: 8, 417001744L: 8
```

Comment:
@mwkrieger , you sure about `416944369L: 8, 417001744L: 8` ? Can you share a Cabana when they show up? Beside those 2 messages, if the Odissey Elite fingerprint includes the existing fingerprint, then we can just use the new one and delete the old one.

Reply:
Sure. Here's a link: https://community.comma.ai/cabana/?route=762a4ddd1129b748|2018-02-19--13-33-13

Regarding the existing fingerprint, there are a number of differences for some reason. 316L is in the EX and not the Elite as well as 304L is in the Elite but not the EX. There are several of these variances which is why I added an Elite specific fingerprint. 

The 416944369L and 417001744L ID's come up when I run the fingerprint method with stock LKAS and ACC engaged. I haven't tried to take them out since OP recognized the Elite.

Reply:
So does this look like it will be upstreamed in the next release? I'd sure like to run 'stock OP' if possible.

Reply:
I'm convinced about the need for 2 fingerprints for Elite and EX, but not about  416944369L and 417001744L :).  not seeing them in the cabana route you shared

Reply:
I just removed the extended fingerprints that were in there. LMK if this looks good.